### PR TITLE
Little ELF segment labelling change ##bin

### PR DESCRIPTION
**Pull request by devnexen**
Name to NULL rather than UNKNOWN where no size.

### CODE REVIEW

**Comment by devnexen** [comment]
One due test failing to fix. Going to ...

File: libr/debug/p/native/linux/linux_coredump.c [hunk 1]
```diff
@@ -44,7 +44,7 @@ static bool is_a_kernel_mapping(const char *name) {
 		&& strcmp (name, "[vvar]")
 		&& strcmp (name, "[heap]")
 		&& strcmp (name, "[vectors]")
-		&& strncmp (name, "[stack", strlen ("[stack")));
+		&& strncmp (name, "[stack", sizeof ("[stack") - 1));
```
Comment (radare) [inline libr/debug/p/native/linux/linux_coredump.c]: sizeof??

Comment (devnexen) [reply libr/debug/p/native/linux/linux_coredump.c]: `        size_t a = sizeof("[stack") - 1;
        size_t b = strlen("[stack");
        printf("%zu %zu\n", a, b);`

Both give 6.

**Comment by radare** [comment]
on widechar strings that wont be correct, we use strlen for this everywhere

**Comment by radare** [comment]
also, rebase and update tests

**Comment by radare** [comment]
i have not much preferences for NULL vs UNKNOWN, but NULL looks more like an error to me

**Comment by devnexen** [comment]
Ok fair enough i can do the change, not an issue to me was just to make the user comfortable with known namings.

**Comment by radare** [comment]
can you make this PR green?

File: libr/debug/p/native/linux/linux_coredump.c (hunk 1 above)
Comment (radare) [reply libr/debug/p/native/linux/linux_coredump.c]: yeah, but sizeof will be affected by the encoding of the string and we use strlen for this case

Comment (devnexen) [reply libr/debug/p/native/linux/linux_coredump.c]: Sorry I forgot to change it back...

**Pull request merged**